README.md

```diff
@@ -2,58 +2,57 @@
 
 Telegram bot for chats administration.
 
+[![Build Status](https://dev.azure.com/GrinderDotNetRu/Grinder/_apis/build/status/Keroosha.grinder?branchName=master)](https://dev.azure.com/GrinderDotNetRu/Grinder/_build/latest?definitionId=1&branchName=master)
+
 ## Configuration and build
 
-To build run you have to add a `bot_config.json` file on path `src\Grinder\`
+Just open project with Rider/VS 2019 and run build
 
-File structure:
+## Running
 
-```
-{
-  "Token": ...,
-  "ChannelId": ...,
-  "AdminUserId": ...,
-  "Socks5Proxy": {
-    "Hostname": "...",
-    "Port": ...,
-    "Username": "...",
-    "Password": "..."
-  },
-  "ChatsToMonitor": [
-    "chat @username",
-    ...
-  ],
-  "AllowedUsers": [
-    "username without at sign",
-    ...
-  ]
-}
-```
+### Using a Docker container
 
-`Token` - bot api token.
+You can just spawn new container with bot from: ***keroosha/grinder*** repo
 
-`ChannelId` - telegram channel id where bot whill write command logs.
+**(DO NOT FORGET ABOUT ENVIRONMENT VARIABLES)**
```

Comment:
So, how am I supposed to "not forget" about them? You give no examples.

Could you please give an example of defining some variables below? Without that, it serves no purpose, right?

Reply:
Yup, I rewrite examples and make a anchor link to config section